nengo_spinnaker/regions/recording.py

```diff
@@ -24,6 +25,21 @@ class WordRecordingRegion(RecordingRegion):
     def bytes_per_frame(self, n_atoms):
         return 4*n_atoms
 
+    def to_array(self, mem, vertex_slice, n_steps):
+        mem.seek(0)
+
+        # Determine how many bytes to read, then read
+        width = vertex_slice.stop - vertex_slice.start
+        framelength = self.bytes_per_frame(width)
+        data = mem.read(n_steps * framelength)
```

Comment:
Could you maybe move some of this logic down into the base class as it's common across all recording regions?